internal/provisioner/plain/README.md

```diff
@@ -85,7 +85,8 @@ my-bundle-instance   my-bundle        my-bundle          InstallationSucceeded
 There is a natural separation between sourcing of the content and application of that content via two separate RukPak
 APIs, `Bundle` and `BundleInstance`. A user can specify a particular `Bundle` to be available in the cluster for
 inspection before any application of the resources. Given a `Bundle` resource named `my-bundle`, the plain provisioner
-will pull down and unpack the bundle to a set of ConfigMaps.
+will pull down and unpack the bundle to a tar.gz file that is saved into a bundle cache directory mounted in the
```

Comment:
Any idea whether we should expose/document that directory location, or do we provide enough discoverability through our set of kustomize manifests?

Reply:
I thought about that. This implementation _does_ make the bundle content a bit less observable. You can no longer `kubectl get configmap -o jsonpath={.data.bundle} | base64 -d | tar zf` or whatever to see individual manifests.

Now its something closer to `kubectl exec <provisionerPod> -- cat /var/cache/bundles/<bundleUID>.tgz | tar tzv`

I think we should handwave over that for now and come up with a better interface.

My initial thought is to define an http api for retrieving bundles and then adding a `ServeHTTP` method to the storage interface such that a provisioner can attach the storage implementation to an http handler, and then put the address of the bundle contents in the bundle status. I'll write another follow-up issue for this idea.

Reply:
> It might be time to also start re-evaluating the objects array on the Bundle's status?

It's like you're in my head. Already working that idea. I should have a PR up today (I think) that removes the entire BundleInfo struct.